source/manage/admin/generating-support-packet.rst

```diff
@@ -185,3 +185,12 @@ The Support Packet contains 3 go runtime profiling files:
 These files can be read using `pprof <https://golang.google.cn/cmd/pprof/>`_.
 
 Use ``go tool pprof -web X`` to open a visualization of the profile in your browser, replacing ``X`` with the profile's file name.
+
+Load metric
+------------
+
+From Mattermost v10.10, system admins have access to an additional diagnostic data point when working with Mattermost support by going to  the **Product** menu |product-list| and selecting **About Mattermost**. 
```

Comment:
This information is not limited to system admins: it's intentionally exposed to any regular user of the system.